spec/models/event_spec.rb

```diff
@@ -180,6 +180,14 @@
     expect(email).to have_attributes(hide_recipients: false, recipients: event.email_adresses_of_rejected_applicants, reply_to: 'workshop.portal@hpi.de', subject: '', content: '')
   end
 
+  it "accepts all its application letters" do
+    event = FactoryGirl.create :event, :with_diverse_open_applications
+    #@event.max_participants = 3
```

Comment:
This is not necessary, since the model function should always work, no matter what max_participants is. I removed it